### PR TITLE
feat(explorer): As a user, I want the Portal page to display latest Attestations

### LINKED ISSUE #783: [US] As a user, I want the Portal page to display latest Attestations
As a user,
I want the Portal page to display latest Attestations,
So that I can get some examples of what can be achieved with this Portal

----

Currently, the [Portal page](https://explorer.ver.ax/linea/portals/0xb86b3e16b6b960fd822849fd4b4861d73805879b) displays the (optional) Modules leveraged by the Portal.

The goal of this feature is to add more interesting data by displaying the last 5 Attestations issued through this Portal.
This can be achieved exactly the same way it's already done on the [Schema page](https://explorer.ver.ax/linea/schemas/0x01f031da36192c34057c764239eb77bb6ec8ebfb808f72a7bb172f37a5bec31f).

### PR DESCRIPTION
**Pull request by Chirag-S-Kotian**
## **Summary**
This PR introduces the `RecentAttestations` component to the Portal page, providing users with an overview of the last 5 attestations issued through the portal. The component mirrors the functionality available on the Schema page, displaying data in a structured, interactive table format. 

---

#### **Key Changes**
1. **New Component: `RecentAttestations`**
   - Fetches and displays the last 5 attestations issued via the portal.
   - Shows a loading skeleton while data is being fetched.
   - Handles error states with an appropriate message.
   - Displays a "No Data Available" message when no attestations exist.

2. **Integration on Portal Page**
   - Added the `RecentAttestations` component to the Portal page layout.
   - Ensures consistent behavior and UI with the Schema page.

3. **Enhanced User Experience**
   - Uses SWR for efficient data fetching and state management.
   - Includes accessibility-friendly error and loading states.
   - Localized strings using `i18n` for global language support.

4. **Robust Error Handling**
   - Gracefully manages fetch errors and empty data scenarios.

---

#### **Testing Instructions**
1. Navigate to the Portal page.
2. Observe the "Recent Attestations" section.
3. Verify the following scenarios:
   - **Data Available**: Displays the last 5 attestations in a DataTable.
   - **Loading State**: Displays a skeleton UI while fetching data.
   - **No Data**: Shows a "No recent attestations available" message if no data exists.
   - **Error State**: Shows an error message when data fetching fails.

4. Check language translation by switching `i18n` locales.

---

#### **Additional Notes**
- Translation keys added for better internationalization:
  - `attestation.recent`
  - `attestation.noData`
  - `error.fetchFailed`

- Component is designed to be reusable and adaptable for other pages if needed.

---

### **Checklist**
- [x] Added `RecentAttestations` component.
- [x] Integrated `RecentAttestations` on the Portal page.
- [x] Implemented robust error and loading state handling.
- [x] Verified localization with `i18n`.
- [x] Tested all scenarios: data available, loading, no data, error.
- [x] Followed coding standards and best practices.

---

### Related ticket

Fixes #783  

### Type of change

- [ ] Chore
- [ ] Bug fix
- [X] New feature
- [ ] Documentation update

## Check list

- [ ] Unit tests for any smart contract change
- [ ] Contracts and functions are documented

### CODE REVIEW

**Comment by alainncls** [comment]
Overall, the PR is good, just a slight change to display recent Attestations _specifically for the Portal_ that we're looking at.
(cf. comments above + add `<RecentAttestations portalId={portal.id} />` to the [Portal page](https://github.com/Consensys/linea-attestation-registry/blob/bc9809b4fe3d6006dc6da6ea9215947560f64ec9/explorer/src/pages/Portal/index.tsx#L95))

File: explorer/src/interfaces/swr/enum.ts [hunk 1]
```diff
@@ -14,4 +14,5 @@ export enum SWRKeys {
   GET_PORTAL_BY_ID = "getPortalByID",
   GET_PORTAL_MODULE_LIST = "getPortalModuleList",
   GET_PORTAL_LIST = "getPortalList",
+  GET_RECENT_ATTESTATION_GLOBAL = "GET_RECENT_ATTESTATION_GLOBAL",
```
[the remark this answers is not in the document]
Comment (Chirag-S-Kotian) [reply explorer/src/interfaces/swr/enum.ts:17]: done

**Review by alainncls — APPROVED** [review]

File: explorer/src/interfaces/swr/enum.ts [hunk 2]
```diff
@@ -6,10 +6,11 @@ export enum SWRKeys {
   GET_RELATED_ATTESTATION = "getRelatedAttestations",
   GET_ATTESTATION_LIST = "getAttestationList",
   GET_ATTESTATION_COUNT = "getAttestationCount",
-  GET_RECENT_ATTESTATION = "getRecentAttestations",
+  GET_RECENT_ATTESTATION_SCHEMA = "getRecentAttestationsSchema",
+  GET_RECENT_ATTESTATION_PORTAL = "getRecentAttestationsPortal",
+  GET_RECENT_ATTESTATION_GLOBAL = "getRecentAttestations",
   GET_MODULE_LIST = "getModuleList",
   GET_MODULE_COUNT = "getModuleCount",
-  SEARCH = "search",
```
Comment (alainncls) [inline explorer/src/interfaces/swr/enum.ts:12]: This key should not be deleted (cf. build job failing)

**Review by alainncls — CHANGES_REQUESTED** [review]

**Review by alainncls — APPROVED** [review]

File: explorer/src/interfaces/swr/enum.ts (hunk 2 above)
Comment (Chirag-S-Kotian) [reply explorer/src/interfaces/swr/enum.ts:12]: done

**Pull request merged**